contrib/kube-prometheus/manifests/grafana/grafana-deployment.yaml

```diff
@@ -27,6 +27,8 @@ spec:
             secretKeyRef:
               name: grafana-credentials
               key: password
+        - name: GF_SERVER_ROOT_URL
```

Comment:
If people prefer accessing Grafana as you say via ingress or node port, it should be a matter of editing the manifests should it not, same as it is for Kibana (in the EFK add-on)?